WcaOnRails/app/models/competition.rb

```diff
@@ -54,6 +55,13 @@ def must_have_at_least_one_delegate
     end
   end
 
+  validate :delegates_must_be_delegates, on: [:update]
+  def delegates_must_be_delegates
```

Comment:
As discussed earlier, could you add a comment explaining the problem here with competitions with former delegates, and link to https://github.com/cubing/worldcubeassociation.org/issues/185#issuecomment-168402252?